content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst

```diff
@@ -1,116 +1,144 @@
-=============================
-How to print shipping labels?
-=============================
+=====================
+Print shipping labels
+=====================
 
-Overview
-========
+.. |DO| replace:: :abbr:`DO (Delivery Order)`
 
-Odoo can handle various delivery methods, including third party shippers
-linked with the transportation company tracking system. It allows you to
-manage the transport company, the real prices and the destination. And
-finally, you will be able to print the shipping labels directly from
-Odoo.
+Integrate Odoo with :doc:`third-party shipping carriers
+<../setup_configuration/third_party_shipper>` to automatically generate shipping labels that
+includes prices, destination addresses, tracking numbers, and barcodes.
 
 Configuration
 =============
 
-Install the shipper company connector module
---------------------------------------------
+To generate labels for a third-party shipping carrier, install the shipping connector, configure and
+activate the delivery method, and provide the company's :ref:`source address
+<inventory/shipping_receiving/configure-source-address>` and :ref:`product weights
+<inventory/shipping_receiving/configure-weight>`.
```

Comment:
@odoo/us-doc-review, I wanted to get your opinion on how short to keep this Configuration section.

Technically, the configuration of this doc is the exact same as the third_party_shipper's (the PR that's supposed to be merged before this one). The only difference is that we set the "integration level" to "Get Rate and Create Shipment" in the delivery method section. 

I chose to keep the shipping connector installation part in this doc because it made sense, but it can be taken out. Should I keep the "Install shipping connector" section in the doc, or should I just link to the other doc?

I decided to not include the source address and product weights configuration in the doc, as this doc and the third_party_shipper one starts to become really similar. Instead, in the "Add shipping on quotation" section, I mention that the source address and product weights must be configured for the shipping connector to work properly. Do you think mentioning it there is enough? Would love to know your thoughts!

Reply:
Hi @Felicious, this is a really good conversation to have– I like that you are trying to reduce writing duplicate information by leveraging the modular nature of docs. After looking at both this PR and #8634, I think that you can remove a lot of the same/similar content you have in this PR (#8658), here is my reasoning:

In the **Configuration** section of #8634, you already detail (with an important admonition) the **Integration Level** needed for generating shipping labels. I think this is a perfect use of an admonition and internal reference.

So for the **Configuration** section of this PR (#8658), I think you can get by with just stating something like the following, removing the **Install shipping connector**, **Delivery method**, and **Production environment** sections from this PR:
```
Configuration
=============

To generate labels for a third-party shipping carrier, first :doc:`install the third party shipping connector
<../setup_configuration/third_party_shipper>`. Then configure and activate the :ref:`delivery method
<inventory-shipping-receiving-configure-delivery-method>`, being sure to set the :guilabel:`Integration
Level` to :guilabel:`Get Rate and Create Shipment` to generate shipping labels. Finally, provide the
company's :ref:`source address <inventory/shipping_receiving/configure-source-address>` and
:ref:`product weights <inventory/shipping_receiving/configure-weight>`. 

.. seealso::
   :doc:`../setup_configuration/third_party_shipper`
```

Lastly, I think your important admonition in the **Add shipping on quotation** works well, and mentioning it there is enough IMO.